examples/microwave-oven-app/linux/main.cpp

```diff
@@ -22,21 +22,17 @@ using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 
-#define MICROWAVE_OVEN_ENDPOINT 1
+static constexpr EndpointId kMicrowwaveOvenEndpointId = 1;
 
-MicrowaveOvenDevice * gMicrowaveOvenDevice = nullptr;
+Platform::UniquePtr<MicrowaveOvenDevice> gMicrowaveOvenDevice;
 
 void ApplicationInit()
 {
-    gMicrowaveOvenDevice = new MicrowaveOvenDevice(MICROWAVE_OVEN_ENDPOINT);
-    gMicrowaveOvenDevice->Init(MICROWAVE_OVEN_ENDPOINT);
+    gMicrowaveOvenDevice = Platform::MakeUnique<MicrowaveOvenDevice>(kMicrowwaveOvenEndpointId);
+    gMicrowaveOvenDevice.get()->Init(kMicrowwaveOvenEndpointId);
```

Comment:
There are 2x `ww` in Microwave --> Rename `kMicrowwaveOvenEndpointId` to `kMicrowaveOvenEndpointId`